README.md

```diff
@@ -169,10 +170,63 @@ ATTR{suspend_mode}=="[oem|keep|off]"
 ```
 
 As en example, to set `suspend_mode`to `off`:
-`ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
+`ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
 `
 
 to create the rule in one line:
 ```shell
-$ echo 'ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
+$ echo 'ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
 ```
+
+### Bypass Charge Control
+
+Bypass Charge can be enabled, disabled, start and stop battery level can be set via sysfs.
+
+n most systems this will be mounted at `/sys/class/power_supply/ayaneo:bypass_charge/` and provides the following files:
+
+#### charge_type
```

Comment:
I like "charge_control_mode" better, to keep it consistent.

Reply:
good point.